### PR TITLE
Add --fully-drawn-magic-string option to startup scenario

### PR DESCRIPTION
**Pull request by akoeplinger**
It can be used on iOS to pass in a magic string that the App is expected to log when it is "fully drawn" and we report the time to that event instead of the normal startup events.

### CODE REVIEW

File: src/scenarios/shared/runner.py
Comment (LoopedBard3) [inline src/scenarios/shared/runner.py:70]: Update the help message here

**Review by LoopedBard3 — APPROVED** [review]
LGTM after help message fix, lets get some test runs before merging.

File: src/scenarios/shared/runner.py
Comment (akoeplinger) [reply src/scenarios/shared/runner.py]: Updated

Comment (akoeplinger) [reply src/scenarios/shared/runner.py]: I don't think this should be called `-delay`, it's the string and not a number of seconds

Comment (LoopedBard3) [reply src/scenarios/shared/runner.py]: ^ I agree. Would it make sense to add the --fully-drawn-extra-delay option as the default startup time delay for iOS?

Comment (TanayParikh) [reply src/scenarios/shared/runner.py]: Ahh okay, I was basing the name change based on the prior description. Yeah definitely doesn't need the `-delay` suffix.

**Comment by LoopedBard3** [comment]
Testing shows that this seems to work and hasn't cause any other problems. Merging

**Pull request merged**
